content/blog/2025/06/2025-06-03-birajit-saikia-gsoc-community-bonding-blog-post.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Meet Birajit Saikia GSoC 2025 Contributor Working on Complete Build Retooling of jenkins.io
+tags:
+  - gsoc
+  - gsoc2025
+  - jenkins.io
+  - jenkins
+  - documentation
+  - tools
+  - development
+  - ui
+authors:
+  - biru-codeastromer
+opengraph:
+  image: /images/gsoc/2025/blogpost_birajit.png
+sig:
+  - gsoc
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+Hello everyone! 
+
+I'm Birajit Saikia, and I’m thrilled to be a Google Summer of Code 2025 contributor with Jenkins. This summer, I’ll be working on the project "Complete Build Retooling of jenkins.io", which focuses on modernizing and reorganizing the Jenkins documentation infrastructure and updating the tech stack with UI/UX changes.
+
+== Project Description
+
+This project focuses on reworking and improving the Jenkins documentation system to make it more structured, modern, and easier to maintain. The documentation will be split between versioned and non-versioned formats using Antora and Gatsby, respectively. Alongside the migration, we’ll also implement improvements in SEO, metadata, site usability, and a modern UI/UX built on an updated tech stack to ensure a better experience for contributors and end users alike.
+
+== Community Bonding Period Works
+
+* Worked on multiple issues to improve documentation, fix broken links on jenkins.io repo.
+- https://github.com/jenkins-infra/jenkins.io/pull/8095[#8095], https://github.com/jenkins-infra/jenkins.io/pull/8103[#8103], https://github.com/jenkins-infra/jenkins.io/pull/8109[#8109], https://github.com/jenkins-infra/jenkins.io/pull/8110[#8110], https://github.com/jenkins-infra/jenkins.io/pull/8113[#8113], https://github.com/jenkins-infra/jenkins.io/pull/8116[#8116], https://github.com/jenkins-infra/jenkins.io/pull/8135[#8135], https://github.com/jenkins-infra/jenkins.io/pull/8136[#8136], https://github.com/jenkins-infra/jenkins.io/pull/8150[#8150], https://github.com/jenkins-infra/jenkins.io/pull/8151[#8151]
+* Created a PR to resolve the React dependency issue: https://github.com/jenkins-infra/docs.jenkins.io/pull/285[#285]
+* Initiated the process of syncing the docs.jenkins.io repository with the latest file structure and content from the jenkins.io repository to ensure consistency and up-to-date information.
+* Had our first weekly meeting on 20th May
```

Comment:
```suggestion
* Had our first weekly meeting on the 20th of May.
```
Small suggestion to add syntax and add full stop so it matches other sentences here